tests/esm-path-mapping/node_modules/ambient/package.json

```diff
@@ -0,0 +1,3 @@
+{
+  "type": "module"
```

Comment:
Node was complaining about ESM `export` syntax in these fake `node_modules`; seems they need their own package.json set to type: module.